internal/environment/ec2.go

```diff
@@ -9,5 +9,10 @@ func (p *EC2) GetPackages() []string {
 }
 
 func (p *EC2) GetServices() []string {
-	return []string{"cloud-init.service"}
+	return []string{
+		"cloud-init.service",
+		"cloud-config.service",
+		"cloud-final.service",
+		"cloud-init-local.service",
+	}
```

Comment:
I'm just curious, if this was really needed (IOW, these services were previously not enabled and should have been) or is it to make this consistent with the `KVM` platform, which also installs and enables `cloud-init`?

Reply:
On the qcow2, they were set on the image config which was inherited by the ami and vhd images.  I don't know if they're necessary, but more importantly I didn't want to change any base definitions.

Reply:
ACK